src/main/org/tvrenamer/model/GlobalOverrides.java

```diff
@@ -19,7 +19,7 @@
     private Map<String, String> showNames;
 
     private GlobalOverrides() {
-        showNames = new HashMap<String, String>();
```

Comment:
Is this the recommended way of doing things? Isn't the purpose of generics to state the type up front and only allow that type? This just seems to negate all the benefits of generics if the type is just inferred by the first entry?